python/vowpalwabbit/sklearn_vw.py

```diff
@@ -94,59 +86,104 @@ def __init__(self,
                  cubic=None,
                  testonly=None,
                  t=None,
+                 holdout_off=None,
+                 holdout_period=None,
+                 holdout_after=None,
+                 early_terminate=None,
+                 passes=1,
+                 initial_pass_length=None,
+                 examples=None,
                  min_prediction=None,
                  max_prediction=None,
                  sort_features=None,
                  loss_function=None,
-                 link=None,
                  quantile_tau=None,
                  l1=None,
                  l2=None,
+                 no_bias_regularization=None,
                  named_labels=None,
                  final_regressor=None,
                  f=None,
                  readable_model=None,
                  invert_hash=None,
                  save_resume=None,
+                 preserve_performance_counters=None,
                  output_feature_regularizer_binary=None,
                  output_feature_regularizer_text=None,
                  oaa=None,
                  ect=None,
                  csoaa=None,
                  wap=None,
+                 probabilities=None,
+                 score=None,
```

Comment:
We should not use score here as in` VWRegressor`  we have score function which is inherited from its parent class so It will cause a problem.

Reply:
oh, good catch